job-utilities/eventbridge_schedule/deploy_schedule.py

```
@@ -48,81 +53,167 @@ def make_schedule_expression(job):
-                  + jobs_file_name + " file")
-            sys.exit(1)
-
-        lambda_client = boto3.client('lambda')
-        try:
-            lambda_details = lambda_client.get_function(
-                FunctionName="job-router-"+environment
-            )
-        except ClientError as e:
-            print("Error getting lambda function: " + e.response['Error']['Code'])
-            sys.exit(1)
+    targets=[]
```

Ruff (0.13.1)

Local variable `targets` is assigned to but never used

Remove assignment to unused variable `targets`

(F841)